string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,10 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  let answer = true;
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toLowerCase()) {
+      answer = false;
+    }
+  }
+  return answer;
+}
```

Comment:
No loop needed! Just compare the original `word` to the whole `word` in lowercase.
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```